src/Castle.Facilities.EventWiring/Castle.Facilities.EventWiring-VS2017.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net45</TargetFramework>
+   
+    <NoWarn>$(NoWarn);1591</NoWarn>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <AssemblyName>Castle.Facilities.EventWiring</AssemblyName>
+    <RootNamespace>Castle.Facilities.EventWiring</RootNamespace>
+    <AssemblyOriginatorKeyFile>../../buildscripts/CastleKey.snk</AssemblyOriginatorKeyFile> 
+    <SignAssembly>true</SignAssembly> 
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <DefineConstants>$(DefineConstants);DOTNET45</DefineConstants>
```

Comment:
There was no reason - I missed it on the main Windsor project. I added it there too.
Regarding `$(DefineConstants)`, it is the recommended way to use `<DefineConstants>$(DefineConstants);MY_CUSTOM_CONSTANT</DefineConstants>` and it keeps the project simple. I noticed that if I remove `$(DefineConstants)`, the TRACE constant goes away. 
I assume the constant DOTNET45 will be removed in the next iterations, does not seem to be needed anymore.